styles.css

```diff
@@ -0,0 +1,61 @@
+body{
```

Comment:
Avoid use of custom css file while using bootstrap. In the majority of the cases bootstrap has classes which fulfils your need. But if you can't find one, always add a reason for adding custom css class.